serverless/src/submitSwodlrOrder/constructOrderPayload.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios'
+
+import { stringify } from 'qs'
+
+import { getEarthdataConfig } from '../../../sharedUtils/config'
+import { getClientId } from '../../../sharedUtils/getClientId'
+import { readCmrResults } from '../util/cmr/readCmrResults'
+/**
+ * Construct the payload that we'll send to Swodlr to create this order
+ */
+export const constructOrderPayload = async ({
```

Comment:
nitpick; Lets rename this method to something more descriptive of what its doing such as `retrieveCMRGranules.js` etc the order payload is really getting set in `handler` which is overriding the variables for the graphql call to their API